docker-compose.apps.yml

```
@@ -190,6 +190,11 @@ services:
       ZOOKEEPER_TICK_TIME: 2000
     ports:
       - "2181:2181"
+    healthcheck:
+      test: [ "CMD", "echo", "ruok", "|", "nc", "localhost", "2181", "|", "grep", "imok" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)